homeassistant/components/minecraft_server/__init__.py

```diff
@@ -0,0 +1,428 @@
+"""The Minecraft Server integration."""
+
+from datetime import timedelta
+import logging
+
+from mcstatus.server import MinecraftServer as MCStatus
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PORT,
+    STATE_UNAVAILABLE,
+    STATE_UNKNOWN,
+)
+from homeassistant.helpers.dispatcher import (
+    async_dispatcher_connect,
+    async_dispatcher_send,
+)
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import (
+    CONF_UPDATE_INTERVAL,
+    DOMAIN,
+    NAME_DESCRIPTION,
+    NAME_LATENCY_TIME,
+    NAME_PLAYERS_LIST,
+    NAME_PLAYERS_MAX,
+    NAME_PLAYERS_ONLINE,
+    NAME_PROTOCOL_VERSION,
+    NAME_VERSION,
+    SIGNAL_NAME_PREFIX,
+)
+
+PLATFORMS = ["sensor"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass, config):
+    """Set up the Minecraft Server component."""
+    return True
+
+
+async def async_setup_entry(hass, config_entry):
+    """Set up Minecraft Server from a config entry."""
+    _LOGGER.debug("Setting up component...")
```

Comment:
I can get rid of those log messages, no problem. But I would like to know why they are not needed? Is it maybe because HA itself prints similar messages (I didn't check, just set my integration to log level "debug")? Thanks.

Reply:
Removed. HA indeed already logs a similar message.